docs/spec.md

```diff
@@ -60,14 +60,16 @@ Each JSON message must be an object containing a field called `op` which identif
   - `parameters`: Allow clients to get & set parameters
   - `parametersSubscribe`: Allow clients to subscribe to parameter changes
   - `time`: The server may publish binary [time](#time) messages
+- `supportedEncodings`: array of strings | undefined, informing the client about which encodings may be used for client side publishing
```

Comment:
It simply means that this field is optional, for backwards compatibility. Is there a better way to mark this field optional?

Reply:
Do we actually want it to be optional long-term, or is this just as a transition? I assume if the server does not send this field, the client will just pick an encoding on its own. That seems like a best-effort fallback rather than an intentional design choice.

If client publishing was not supported at all I imagine `supportedEncodings` would not appear at all, rather than be set to `undefined`. What about rewording as: "`string[]`, informing the client about which encodings may be used for client-side publishing. Only set if client publishing is supported"

Reply:
:+1: 

```suggestion
- `supportedEncodings`: array of strings | informing the client about which encodings may be used for client-side publishing. Only set if client publishing is supported.
```